<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fm_VendasGerais</class>
 <widget class="QDialog" name="Fm_VendasGerais">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="lbl_from">
               <property name="text">
                <string>De:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="de_from"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="lbl_to">
             <property name="text">
              <string>At√©</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="de_to"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btt_filter">
         <property name="text">
          <string>Filtrar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btt_filterUndo">
         <property name="text">
          <string>Sem Filtro</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tw_sellings"/>
       </item>
       <item>
        <widget class="QListWidget" name="tw_sellItems"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="btt_generatePDF">
           <property name="text">
            <string>Gerar PDF</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
